npm_and_yarn/spec/dependabot/npm_and_yarn/file_updater/npm_lockfile_updater_spec.rb

```diff
@@ -167,6 +168,33 @@
           to include("19c4dba3bfce7574e28f1df2138d47ab4cc665b3")
       end
     end
+
+    context "when the packages name needs sanitizing" do
+      let(:files) { project_dependency_files("npm7/simple") }
+
+      it "restores the packages name attribute" do
+        parsed_lockfile = JSON.parse(updated_npm_lock_content)
+        expect(parsed_lockfile.dig("packages", "", "name")).to eq("{{ name }}")
```

Comment:
This is sad, we currently re-write this to `something` because https://github.com/dependabot/dependabot-core/blob/f7d4608eed73741fa0d0a759f7e93bf74b9d2c9e/npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/npm_lockfile_updater.rb#L786